app/services/resource_service.rb

```diff
@@ -12,78 +12,126 @@ def initialize(data_provider:)
   def create(resource_class, params)
     @params = params
 
+    # cleanup params for given :id
+    @old_resource_id = @params.delete(:id)
```

Comment:
in den `@params` selbst wird die `id` nicht mehr benötigt?

Reply:
nein, im Gegenteil: eine ID läßt sich nicht verändern